packages/wdio-devtools-service/src/auditor.js

```diff
@@ -0,0 +1,131 @@
+import weightedMean from 'weighted-mean'
+
+import Diagnostics from 'lighthouse/lighthouse-core/audits/diagnostics'
+import MainThreadWorkBreakdown from 'lighthouse/lighthouse-core/audits/mainthread-work-breakdown'
+import Metrics from 'lighthouse/lighthouse-core/audits/metrics'
+import TTFBMetric from 'lighthouse/lighthouse-core/audits/time-to-first-byte'
+
+import { quantileAtValue } from './utils'
+import logger from '@wdio/logger'
+
+const log = logger('Auditor')
+
+/**
+ * metric scoring to calculate Lighthouse Performance Score
+ * contains: [media, falloff]
+ * see https://docs.google.com/spreadsheets/d/1_iDW0B870vZF6dAhf1Icdher0gX_KFxCd6Df0_fZ6do/edit#gid=283330180
+ */
+const METRIC_SCORING = {
+    FMP: [4000, 1600], // first meaningful paint
```

Comment:
Are these FMP, FI, etc described somewhere? (maybe in doc above)

Reply:
they come from the document described above